kernel/Kconfig

```diff
@@ -327,6 +327,14 @@ config BOOT_DELAY
 	  achieved by waiting for DCD on the serial port--however, not
 	  all serial ports have DCD.
 
+config BOOT_DELAY_RANDOM
+	bool "Randomize boot delay"
+	default n
```

Comment:
No need to have `default n` as the option is disabled by default.

Reply:
done